ta_services/ta_services.mk

```diff
@@ -0,0 +1,43 @@
+.PHONY: all
+all: ta_dev_kit
+
+.PHONY: clean
+clean:
```

Comment:
Sorry for late reply but I just saw this. Lines 1-5 look like some stray code?

Reply:
Will fix it now